ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/snmp-settings.j2

```diff
@@ -1,9 +1,24 @@
 {% if snmp_server is defined and snmp_server is not none %}
 ### SNMP Configuration Summary
 
-| Contact | Location | SNMP Traps | IPv4 ACL | IPv6 ACL |
-| ------- | -------- | ---------- | -------- | -------- |
-| {{ snmp_server.contact | default('-') }} | {{ snmp_server.location | default('-') }} | {% if snmp_server.traps.enable is defined and snmp_server.traps.enable is not none %}{% if snmp_server.traps.enable %} Enabled {% else %} Disabled {% endif %}{% endif %} | {{ snmp_server.access_list_v4 | default('-') }} | {{ snmp_server.access_list_v6 | default('-') }} |
+| Contact | Location | SNMP Traps |
+| ------- | -------- | ---------- |
+| {{ snmp_server.contact | default('-') }} | {{ snmp_server.location | default('-') }} | {% if snmp_server.traps.enable is defined and snmp_server.traps.enable is not none %}{% if snmp_server.traps.enable %} Enabled {% else %} Disabled {% endif %}{% else %} Disabled {% endif %} |
```

Comment:
Double else?

Reply:
Yes. Probably there is a better way to optimise but:
if snmp_server.traps.enable exists and defined:
   if snmp_server.traps.enable equals TRUE:
      print Enabled
   else //Variable is defined and FALSE
      print Disabled
   endif
else //From the first if. Basically, if the variable is not defined or empty
print Disabled
endif

Agree?